pkgs/development/python-modules/std2/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+}:
+
+buildPythonPackage {
+  pname = "std2";
+  version = "unstable-2023-07-05";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "ms-jpq";
+    repo = "std2";
+    rev = "1080b7b657f75352481808a906939d16b6d956a1";
+    fetchSubmodules = false;
+    hash = "sha256-shAuCZvyFGrW26uTnNeyvlAIABb1d9oeKpk9hQbtuEQ=";
+  };
+
+  propagatedBuildInputs = [ setuptools ];
```

Comment:
is this required?

Reply:
Yes, it doesn't build without.
It only works with both `format = "pyproject"` and `propagatedBuildInputs = [ setuptools ];`.

Reply:
works for me after moving setuptools to `nativeBuildIputs`

Reply:
Indeed ! I changed it in both packages.